docs/getting-started/create-pipeline.md

```diff
@@ -6,11 +6,6 @@ Using the Screwdriver CLI, you can build pipelines locally on your machine.
 **The Screwdriver CLI does not exist yet, but here is what the instructions would look like.**
 
 ### Installing Go
```

Comment:
I know this is not the piece you were changing, but all of this stuff is very mac-specific. I feel like in many cases we should just link to official documentation instead of trying to give specific advice that might change over time.

It should be sufficient to say "Go 1.6 or higher is required. Go here to find out how to setup your go environment:"